org.gimp.GIMP.json

```diff
@@ -645,15 +645,9 @@
             "name": "luajit",
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz",
-                    "sha256": "1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3",
-                    "x-checker-data": {
-                        "type": "anitya",
-                        "project-id": 6444,
-                        "stable-only": false,
-                        "url-template": "https://luajit.org/download/LuaJIT-$version.tar.gz"
-                    }
+                    "type": "git",
+                    "url": "https://github.com/LuaJIT/LuaJIT.git",
+                    "branch": "v2.1"
```

Comment:
you NEVER build a branch. This break reproducibility.

Reply:
That's what I was questioning in https://github.com/flathub/org.gimp.GIMP/pull/344#issuecomment-2295348346. Thanks for the response.